config/config.py.template

```diff
@@ -429,6 +429,32 @@ MLLIB_GLM_REGRESSION_TEST_OPTS = MLLIB_GLM_TEST_OPTS + [
 MLLIB_TESTS += [("glm-regression", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
     MLLIB_JAVA_OPTS, [ConstantOption("glm-regression")] + MLLIB_GLM_REGRESSION_TEST_OPTS)]
 
+if MLLIB_SPARK_VERSION >= 1.5:
+    MLLIB_GLM_ELASTIC_NET_REGRESSION_TEST_OPTS = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        # Loss to minimize: l2 (squared error)
+        OptionSet("loss", ["l2"]),
+        # The max number of iterations for LBFGS/OWLQN
+        OptionSet("num-iterations", [20]),
+        # LBFGS/OWLQN is used with elastic-net regularization.
+        OptionSet("optimizer", ["lbfgs"]),
+        # Using elastic-net regularization.
+        OptionSet("reg-type", ["elastic-net"]),
+        # Runs with L2 (param = 0.0), L1 (param = 1.0), and mixing L1/L2.
+        OptionSet("elastic-net-param", [0.0, 0.5, 1.0]),
+        # Runs with lambda = [0.0, 0.5]
```

Comment:
nit: I would prefer if this was just 0.5 to keep perf-test parameter space small